solutions/client-project-setup/project-iam.yaml

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #########
-# First, Grant GCP Role IAM service account admin on project to client-config-control SA so that it can create service accounts.
+# Grant GCP Role IAM service account admin on project to client-config-control SA so that it can create service accounts.
+# AC-3(7), AC-3, AC-16(2) - This service account possesses limited privileges(permissions) and is restricted to performing only the necessary operations for resources within the designated namespace. The service account is associated with the namespace and is assigned roles as required.
```

Comment:
all the security controls - per yaml in addition to consolidated securitycontrols.md are appreciated - and primed for automated parsing